### PR TITLE
Fix/packaging

### PR DESCRIPTION
**Pull request by colinsurprenant**

### CODE REVIEW

**Comment by ph** [comment]
Working on macos all the gems are there, doing a quick run in windows before merging in 1.5 and master.

File: rakelib/artifacts.rake [hunk 1]
```diff
@@ -43,7 +43,17 @@ namespace "artifact" do
     end.flatten.uniq
   end
 
-  task "prepare" => ["bootstrap", "plugin:install-default"]
+  task "use-defaults-gemfile" do
+    FileUtils.cp("Gemfile.defaults", "Gemfile")
```
Comment (jordansissel) [inline rakelib/artifacts.rake]: I might be missing context, so bear with me - This file copy feels... weird. Is the ~/Gemfile not important anymore?

Comment (jordansissel) [inline rakelib/artifacts.rake]: Alternately, if it's only part of package building, we don't have to overwrite anything, we can make the "copy" happen during packaging.

Comment (ph) [inline rakelib/artifacts.rake]: The copy is only done when calling a task of the `artifact:*` family.
The freezing is currently a human process before an official release.

Comment (jordansissel) [inline rakelib/artifacts.rake]: I'm OK if this stays for RC1. We can investigate a better solution for GA :)

**Comment by ph** [comment]
I redid all the users test on windows with a zip generated from this PR with great success.
:shipit:

**Comment by ph** [comment]
@colinsurprenant We don't need to include the *.default files in a release since they are only used in the packaging process.

**Comment by ph** [comment]
tested on macos too working fine, merging this.

**Comment by colinsurprenant** [comment]
@ph that's right, the `*.default` files do not need to be included in the package, just checked-in the branch.

**Pull request closed without merging**